web/tests/libtest/codechecker.py

```diff
@@ -489,19 +505,18 @@ def start_or_get_server():
             # 'Default' product in SQLite mode, if the server was started
             # brand new. But certain test modules might make use of a
             # default product, so we now manually have to create it.
-            print("PostgreSQL server does not create 'Default' product...")
-            print("Creating it now!")
-            default_path = os.path.join(config_dir, 'Default')
+            default_path = os.path.join(workspace_root, 'Default')
             try:
+                print("PostgreSQL server does not create 'Default' product...")
+                print("Creating it now!")
+
                 os.makedirs(default_path)
+                add_test_package_product({'viewer_host': 'localhost',
```

Comment:
Why did you move `add_test_package_product` in the try catch block?
It will only catch OSError but `add_test_package_product` probably will not raise that type of exception. 
In `add_test_package_product` probably some connection errors will happen if the server is not started yet.

Reply:
Yes, it will not raise an exception, but the `os.makedirs` function will raise an exception if the directory exists so the `add_test_package_product` will not be called in this case.

I've replaced the try/except block with an if block where I check the existence of this directory 
 with the `os.path.exists` function.